### PR TITLE
Mesos validation

### PR DESCRIPTION
**Pull request by sherzberg**
validate whether or not a MESOS_TASK_ID is running on a mesos slave

### CODE REVIEW

File: builtin/credential/marathon/backend_test.go [hunk 1]
```diff
@@ -21,20 +22,36 @@ func TestBackend_basic(t *testing.T) {
...
+		},
+	})
+}
+
 func testAccPreCheck(t *testing.T) {
 	if v := os.Getenv("MARATHON_URL"); v == "" {
 		t.Fatal("MARATHON_URL must be set for acceptance tests")
 	}
+	if v := os.Getenv("MESOS_URL"); v == "" {
+		t.Fatal("MARATHON_URL must be set for acceptance tests")
```
Comment (DustinChaloupka) [inline builtin/credential/marathon/backend_test.go]: `MESOS_URL`

**Comment by DustinChaloupka** [comment]
Other than that one line that could confuse on why it's not working. :+1:

**Comment by lamdor** [comment]
:+1: This looks good to me. Do we have anything that checks that the request source IP matches one of the mesos slaves?

**Comment by sherzberg** [comment]
@rubbish im not sure that is going to be possible from big. The RemoteAddr from the request gives me the docker gateway (172.17.42.1) when running with big.

My hope is that in env-setup, and net=host, that I will be able to get the correct IP. If that's true, I'll circle back.

**Pull request merged**